data/json/furniture_and_terrain/terrain-flora.json

```diff
@@ -2055,9 +2055,9 @@
     "id": "t_fern",
     "name": "fern",
     "description": "A short plant with thousands of leaves which grow in a fractal pattern from the stem.",
-    "symbol": "#",
+    "symbol": ",",
     "looks_like": "t_underbrush",
-    "color": [ "green", "brown_green", "brown", "brown" ],
+    "color": [ "brown_green", "green",  "green", "green" ],
```

Comment:
The pull request checker dislikes the additional space here.

Reply:
I was at a loss as to what was wrong.
Wasn't there extra space?
Thank you very much.